src/SDK.ts

```diff
@@ -938,7 +969,9 @@ export default class SDK implements ISDK {
       } catch (error) {
         const elapsedTimeInMilliseconds = timer.milliSecondsElapsed;
         this.logWithLogger(LogLevel.ERROR, OCSDKTelemetryEvent.FETCHDATAMASKINGFAILED, "Fetch Data Masking Failed", requestId, undefined, elapsedTimeInMilliseconds, requestPath, method, error);
-
+        if (error.code === Constants.axiosTimeoutErrorCode) {
+          throwClientHTTPTimeoutError();
+        }
```

Comment:
this 3 lines are repeated a lot of times, if in the future we need to modify it, we willneed to go over this in all the places , with the risk of missing one, 

could we use a function and just invoke it?, so for future modification will be in a single place, 

DRY.